core/src/datayoga_core/block.py

```diff
@@ -93,6 +95,22 @@ def stop(self):
     def get_block_name(self):
         return os.path.basename(os.path.dirname(sys.modules[self.__module__].__file__))
 
+    def produce(self) -> Generator[Message, None, None]:
```

Comment:
should we move to Producer altogether and separate from Block?

Reply:
Probably we should split them completely. May be next time ?